### PR TITLE
add initial python36 implementation

### PR DESCRIPTION
**Pull request by elmiko**
this change creates a new build path for a python36 based pyspark image.

* add makefile for py36 pyspark
* add new makefile to root makefile
* add concreate image definition for py36
* add concreate generated build files
* add templates for py36 specific builds

### CODE REVIEW

**Comment by elmiko** [comment]
this change contains only the build configs and artifacts for python36, it does not have tests. those are coming in a followup

File: image.pyspark-py36.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+schema_version: 1
+
+name: "radanalyticsio/radanalytics-pyspark-py36"
+version: 1.0
+release: dev
+from: "centos/python-36-centos7:latest"
+labels:
+  - name: "io.k8s.description"
+    value: "Platform for building a radanalytics pyspark app"
```
Comment (tmckayus) [inline image.pyspark-py36.yaml]: Do we want to include 36 in this description too, or do you think that's overkill?

Comment (elmiko) [reply image.pyspark-py36.yaml]: i think that would be helpful, if only so folks know what they made

File: image.pyspark-py36.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      - java-1.8.0-openjdk
+artifacts:
+  - url: https://archive.apache.org/dist/spark/spark-2.3.0/spark-2.3.0-bin-hadoop2.7.tgz
+    md5: af45eeb06dc1beee6d4c70b92c0e0237
+  - url: https://github.com/radanalyticsio/oshinko-cli/releases/download/v0.5.3/oshinko_v0.5.3_linux_amd64.tar.gz
+    md5: 1995c4eb4013c1b4b1750dd3299e2e30
+run:
+    user: 185
+    cmd:
+      - /usr/libexec/s2i/usage
```
Comment (tmckayus) [inline image.pyspark-py36.yaml]: We have the bootstrap.sh included below in the build dir, but it's not being called out here in the entrypoint (see the image.pyspark.yaml)

File: templates/python36build.json [hunk 3]
```diff
@@ -0,0 +1,117 @@
+{
```
Comment (tmckayus) [inline templates/python36build.json]: Looks like this is missing recent template changes in a diff against pythonbuild.json -- addition of lookupPolicy

File: templates/python36builddc.json [hunk 4]
```diff
@@ -0,0 +1,318 @@
+{
```
Comment (tmckayus) [inline templates/python36builddc.json]: Looks like this is missing recent template changes in a diff against pythonbuilddc.json

addition of lookupPolicy
addition of kube scheduler param and env var
addition application_name env var

File: image.pyspark-py36.yaml (hunk 2 above)
Comment (elmiko) [reply image.pyspark-py36.yaml]: i see the error, something got chopped when i rebased recently

**Comment by elmiko** [comment]
ok, updated to address the issues. thanks @tmckayus !

**Comment by tmckayus** [comment]
lgtm

**Pull request merged**